magento/rest.py

```diff
@@ -13,10 +14,24 @@ def __init__(self, url, token, verify_ssl=True):
         self._token = token
         self._verify_ssl = verify_ssl
 
-    def call(self, resource_path, arguments):
+    def call(self, resource_path, arguments, http_method=None, storeview=None):
         url = '%s/%s' % (self._url, resource_path)
-        res = requests.get(
-            url, params=arguments, verify=self._verify_ssl,
-            headers={'Authorization': 'Bearer %s' % self._token})
+        if storeview:
+            # https://github.com/magento/magento2/issues/3864
+            url = url.replace('/rest/V1/', '/rest/%s/V1/' % storeview)
```

Comment:
I could do that, but I thought that this was the right place in the abstraction layer (especially because at the time this aspect of the API was undocumented). But we can refactor on our side.